Godeps

```diff
@@ -83,14 +84,18 @@ github.com/wvanbergen/kazoo-go 968957352185472eacb69215fa3dbfcfdbac1096
 github.com/yuin/gopher-lua 66c871e454fcf10251c61bf8eff02d0978cae75a
 github.com/zensqlmonitor/go-mssqldb ffe5510c6fa5e15e6d983210ab501c815b56b363
 golang.org/x/crypto dc137beb6cce2043eb6b5f223ab8bf51c32459f4
-golang.org/x/net f2499483f923065a842d38eb4c7f1927e6fc6e6d
+golang.org/x/net cbe0f9307d0156177f9dd5dc85da1a31abc5f2fb
```

Comment:
Is the version change to `golang.org/x/net` required?

Reply:
I'm unable to build the project using the existing `golang.org/x/net` version - seems the update is required by `google.golang.org/grpc`